src/components/App.js

```diff
@@ -7,6 +7,7 @@ import {
 } from '@material-ui/core';
 import ButtonPanel from './ButtonPanel';
 import Display from './Display';
+// import calculate from '../logic/calculate';
```

Comment:
## Commented out code
- [ ] Please uncomment this before moving ahead. I understand the implications it brings but this milestone only checks for the implemented logic and it is not expected to run properly, thank you!